tutorials/irpa-ui-automation-recorder/irpa-ui-automation-recorder.md

```diff
@@ -0,0 +1,190 @@
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 4: ](Record the Application - Part 1)]
+
+> The UI Automation Recorder records the steps you perform across the screens of an application.
+ Whenever the screen (DOM) is changed in the provider application, you must capture it by choosing the **Record** button in the recording widget.
+
+1.  Click the **Record** !![record](record.png) button to start recording in recorder widget.
+
+2.  Perform a simple calculation by adding `10` and `5` to get result `15`. Click `10`, `+`, `5` and `=`. You will notice that the steps are getting recorded in the recorder.
+
+3.  Click the result `15` in Calculator so that it gets recorded too. You will later use this result in the automation.
+
+    !![recorder calculation simple](recorder_standard.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 5: ](Record the Application - Part 2)]
+
+1.  Click **Open Navigation** icon !![Open Navigation](open_navigation.png).
+
+2.  There is a change in DOM so, you must record this screen. Click the **New Capture** !![capture icon](capture_icon.png) button to record the new screen.
+    A new screen appears in the recorder.
+
+    !![recorder navigation](recorder_navigation.png)
+
+3.  From the left navigation bar in Calculator, click **Time**.
+
+    !![calculator navigation](calculator_navigation.png)
+
+4.  Click the **New Capture** !![capture icon](capture_icon.png) button to record the new screen.
+
+    !![calculator hour](calculator_hour.png)
+
+5.  You need to convert **Minutes** to **Seconds**. Click the first dropdown and capture the new screen with the dropdown elements.
+    Click **Seconds**.
+
+    !![calculator dropdown](calculator_dropdown.png)
+
+6.  The Calculator shows **Seconds** and **Minutes**. Record this screen in recorder widget.
+
+7.  Select `0` value for **Minutes** and then, type `10`.
+
+    !![calculator minutes to seconds](calculator_minutes_to_seconds.png)
+
+8.  Select the resultant value `600` in **Seconds**. You will use this value later in automation. You will see the following steps added to the recorder widget.
+
+    !![recorder complete](recorder_complete.png)
+
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 6: ](Export the Recording)]
+
+1. After completing all the steps, click the **Stop** !![stop](stop_icon.png) button to stop the recording.
+
+    !![recorder final](recorder_final.png)
+
+2.  Click **Export** to move the recording to the Cloud Studio.
+
+    !![recorder export](recorder_export.png)
+
+3.  Close the Calculator application.
+
+[DONE]
+[ACCORDION-END]
+
+[ACCORDION-BEGIN [Step 7: ](Review the Automation)]
+
+> Once all the artifacts are moved to the Cloud Studio, the automation can be executed to replay the recorded steps. It can also be modified and then executed from the Cloud Studio.
+
+1.  You will see all the screens and steps are recorded in the automation in the Cloud Studio.
```

Comment:
This is no user action. Exclude it from the list.

Reply:
Done